### PR TITLE
Feature/rickwoltheus/taalhuis student update

### PR DESCRIPTION
**Pull request by RickWoltheus**
this pr adds the update query to the participant update page

### CODE REVIEW

File: client/src/views/Authorized/Participants/taalhuis/Participants/Detail/Intake/ParticipantIntakeView.tsx [hunk 1]
```diff
@@ -70,7 +70,6 @@ export const ParticipantsIntakeView: React.FunctionComponent<Props> = props => {
                 </TabSwitch>
                 {renderSection()}
             </Column>
-            <Space pushTop={true} />
```
Comment (Grkmk) [inline client/src/views/Authorized/Participants/taalhuis/Participants/Detail/Intake/ParticipantIntakeView.tsx:73]: unused import console warning?

**Review by Grkmk — CHANGES_REQUESTED** [review]
just looking at the code, you probably have console warnings

**Review by Grkmk — CHANGES_REQUESTED** [review]
Screen Shot 2021-04-01 at 5 53 32 PM

**Comment by RickWoltheus** [comment]
Screen Shot 2021-04-01 at 5 53 32 PM

Wot how

**Comment by Grkmk** [comment]
> Screen Shot 2021-04-01 at 5 53 32 PM
> 
> Wot how

are you checking your console? try stopping & rerunning the script if you dont have iterm's keep logs setting on. otherwise cra might clear warnings on re-compile

**Comment by RickWoltheus** [comment]
@Grkmk  it should be fixed now

**Review by Grkmk — APPROVED** [review]
🏅

**Pull request merged**